sdk/include/reactos/version.rc

```diff
@@ -19,7 +19,7 @@
 #define REACTOS_DEFAULT_STR_COMPANY_NAME    "ReactOS Project"
 #define REACTOS_DEFAULT_STR_DESCRIPTION     "ReactOS Core Component"
 #define REACTOS_DEFAULT_STR_INTERNAL_NAME   ""
-#define REACTOS_DEFAULT_STR_LEGAL_COPYRIGHT "Copyright 1996-" COPYRIGHT_YEAR " ReactOS Project"
+#define REACTOS_DEFAULT_STR_LEGAL_COPYRIGHT "1996-" COPYRIGHT_YEAR " ReactOS Project"
```

Comment:
You are missing the copyright in this string now, which was present in ours and is present in Microsoft's version.
Either use the Copyright-symbol, or leave the text.

Reply:
I will use "(C) " or the single Copyright-character that MS uses to satisfy you. Although I do think that both is redundant. See the pics above.

Reply:
Some Asian font has no copyright symbol.

Reply:
Thank you Katayama. So it will become "(C) ".

Reply:
'To satisfy you', and not to follow the actual MSDN description?
Sure.

> Copyright notices that apply to the file. This should include the full text of all notices, legal symbols, copyright dates, and so on. This string is optional.

https://learn.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource

Reply:
(c) restored.